src/lib/Menu/ContentEditAnchorMenuBuilder.php

```diff
@@ -0,0 +1,165 @@
+
+        /** @var array<string, array<string>> $groupedFields */
+        $groupedFields = $options['grouped_fields'];
+
+        $items = [
+            self::ITEM__CONTENT => $this->createMenuItem(
+                self::ITEM__CONTENT,
+                [
+                    'attributes' => ['data-target-id' => 'ibexa-edit-content-sections-content-fields'],
+                    'extras' => [
+                        'orderNumber' => 10,
+                    ],
+                ]
+            ),
+        ];
+
+        $items[self::ITEM__CONTENT]->setChildren(
+            $this->getContentFieldGroupItems($groupedFields)
+        );
+
+        $metaFields = $this->getMetaFieldItems($contentType);
+
+        if (!empty($metaFields)) {
+            $items[self::ITEM__META] = $this->createMenuItem(
+                self::ITEM__META,
+                [
+                    'attributes' => ['data-target-id' => 'ibexa-edit-content-sections-meta'],
+                    'extras' => [
+                        'orderNumber' => 50,
+                    ],
+                ]
+            );
+
+            $items[self::ITEM__META]->setChildren($metaFields);
+        }
+
+        $menu->setChildren($items);
+
+        return $menu;
+    }
+
+    /**
+     * @param array<string, array<string>> $groupedFields
+     *
+     * @return array<\Knp\Menu\ItemInterface>
+     */
+    private function getContentFieldGroupItems(array $groupedFields): array
+    {
+        $items = [];
+        $order = 0;
+        foreach ($groupedFields as $group => $fields) {
+            $order = $order + 10;
+            $items[$group] = $this->createMenuItem($group, [
+                'attributes' => [
+                    'data-target-id' => sprintf('ibexa-edit-content-sections-content-fields-%s', mb_strtolower($group)),
+                ],
+                'extras' => [
+                    'orderNumber' => $order,
+                ],
+            ]);
+        }
+
+        return $items;
+    }
+
+    /**
+     * @return array<\Knp\Menu\ItemInterface>
+     */
+    private function getMetaFieldItems(ContentType $contentType): array
+    {
+        $fieldTypeSettings = $this->configResolver->getParameter('admin_ui_forms.content_edit.fieldtypes');
+        $metaFieldTypeIdentifiers = array_keys(array_filter(
+            $fieldTypeSettings,
+            static fn (array $config) => true === $config['meta']
+        ));
+
+        $items = [];
+        $order = 0;
+        foreach ($metaFieldTypeIdentifiers as $metaFieldTypeIdentifier) {
+            if (false === $contentType->hasFieldDefinitionOfType($metaFieldTypeIdentifier)) {
+                continue;
+            }
+
+            $fieldDefinitions = $contentType->getFieldDefinitionsOfType($metaFieldTypeIdentifier);
+            foreach ($fieldDefinitions as $fieldDefinition) {
+                $fieldDefIdentifier = $fieldDefinition->identifier;
+                $order = $order + 10;
```

Comment:
`10` could be moved to a constant as it appears in several places within the class.